nixos/modules/services/security/clamav-unofficial-sigs.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.clamav-unofficial-sigs;
+in {
+  options.services.clamav-unofficial-sigs = with types; {
```

Comment:
Should this live under `services.clamav` which already exists? I'm not familiar with this software so let me know what you think 🤔

Reply:
The option could also be moved there, I just thought I'd create a separate module and option because the signatures are not from any official source.

This service can be seen as extension to freshclam. It uses other malware signature sites (possibly paid ones with credentials) and adds the signatures to the clamav database.